docs/source/core_concepts/glossary.rst

```diff
@@ -259,12 +259,15 @@ Transaction Status Set
 ^^^^^^^^^^^^^^^^^^^^^^
 
  - NOT_RECEIVED: requested peer does not have this transaction.
```

Comment:
Please, arrange the statuses, so they would go from "initial" to "final" ones. For example, MST_EXPIRED should be in the end along with COMMITTED or REJECTED

Reply:
Do not agree. This is arranged with respect to this diagram, which shows transactions pipeline and statuses are respective outcomes at each validation step:

<img src="https://github.com/hyperledger/iroha/blob/4e68ef5ecb85bc72f89adac4b727ebaf7c029776/docs/image_assets/tx_status.png?raw=true" />